manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```diff
@@ -24,3 +24,9 @@ data:
   ## Feature stage:
   ## [Alpha](https://github.com/kubeflow/pipelines/blob/07328e5094ac2981d3059314cc848fbb71437a76/docs/release/feature-stages.md#alpha)
   cronScheduleTimezone: "UTC"
+
+  ## cacheImage is the image that the mutating webhook will use to patch
+  ## cached steps with. Will be used to echo a message announcing that 
+  ## the cached step result will be used. If not set it will default to 
+  ## 'gcr.io/google-containers/busybox@sha256:545e6a6310a27636260920bc07b994a299b6708a1b26910cfefd335fdfb60d2b'
+  cacheImage: "gcr.io/google-containers/busybox@sha256:545e6a6310a27636260920bc07b994a299b6708a1b26910cfefd335fdfb60d2b"
```

Comment:
discussion: shall we pin to a specific version?

pros for not pinning:
* we don't need to update it from time to time because of CVEs

cons for not pinning:
* users may get new images from time to time, so image cache hit rate could be lower (but users can configure it to pin a version though)
* there might be a risk the latest image will break us, but I think that's low risk considering we just run an echo command there.

I'm leaning towards not pinning a version to reduce our maintenance cost.

What do you think?

/cc @capri-xiyue @Ark-kun

Reply:
For this container we need to pin a version, there is no latest and will fail if version is not specified, we can switch to`gcr.io/google-containers/busybox` and then we don't need a version but it is slightly larger, took the smallest of the two.  

Since we only use `echo`, do we really expect to have to upgrade this image in the future and use latest? 

Also sorry for the first version, mixing the env names all over the place and made a mess of the changes in the manifests @Bobgy